pkg/api/api.go

```diff
@@ -183,8 +183,7 @@ func (a *API) newRoute(path string, handler http.Handler, isPrefix, auth, gzip b
 	return route
 }
 
-// RegisterAlertmanager registers endpoints associated with the alertmanager. It will only
-// serve endpoints using the legacy http-prefix if it is not run as a single binary.
+// RegisterAlertmanager registers endpoints associated with the alertmanager.
```

Comment:
```suggestion
// RegisterAlertmanager registers endpoints that are associated with the alertmanager.
```

Reply:
Easier to understand when the phrase does not have to be filled in by the reader